main.ts

```diff
@@ -414,6 +414,10 @@ class Translator {
 
       case ts.SyntaxKind.EnumDeclaration:
         var decl = <ts.EnumDeclaration>node;
+        // The only legal modifier for an enum decl is const.
```

Comment:
Could you be explicit about this containing const? It might change in the future, and it's also more readable. I think "export enum Foo" might also be a thing.